### PR TITLE
add upgrade instruction in cad

### PR DESCRIPTION
**Pull request by 0xaptosj**
### Checklist

- Do all Lints pass?
  - [ ] Have you ran `pnpm spellcheck`?
  - [ ] Have you ran `pnpm fmt`?
  - [ ] Have you ran `pnpm lint`?

### CODE REVIEW

File: apps/docusaurus/docs/create-aptos-dapp/templates/digital-asset.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ Create or find the account you want to create a collection with. If you haven't
 
 3. Run `npm run move:publish`
 
+### Upgrade the Move module
+
+When you make change to your Move module, there are 2 kinds of changes, compatible and incompatible. You can learn [upgrade policy in details on docs](https://aptos.dev/move/book/package-upgrades/#compatibility-rules). When you make compatible changes, you run `npm run move:upgrade`. When you make incompatible changes, you run `npm run move:publish` to publish to a new address.
```
Comment (hariria) [inline apps/docusaurus/docs/create-aptos-dapp/templates/digital-asset.md:63]:
```suggestion
When you make changes to your Move module, there are 2 kinds of changes, compatible and incompatible. You can learn [upgrade policy in details on docs](https://aptos.dev/move/book/package-upgrades/#compatibility-rules). When you make compatible changes, you can run `npm run move:upgrade` <TO DO WHAT?>. When you make incompatible changes, you can run `npm run move:publish` to publish to a new address.
```

File: apps/docusaurus/docs/create-aptos-dapp/templates/fungible-asset.md [hunk 2]
```diff
@@ -58,6 +58,10 @@ Create or find the account you want to create an asset with. If you haven't crea
 
 3. Run `npm run move:publish`
 
+### Upgrade the Move module
+
+When you make change to your Move module, there are 2 kinds of changes, compatible and incompatible. You can learn [upgrade policy in details on docs](https://aptos.dev/move/book/package-upgrades/#compatibility-rules). When you make compatible changes, you run `npm run move:upgrade`. When you make incompatible changes, you run `npm run move:publish` to publish to a new address.
```
Comment (hariria) [inline apps/docusaurus/docs/create-aptos-dapp/templates/fungible-asset.md:63]: See above

File: apps/nextra/pages/en/build/create-aptos-dapp/templates/digital-asset.mdx [hunk 3]
```diff
@@ -73,6 +73,10 @@ npm run move:init
 npm run move:publish
 ```
 
+### Upgrade the Move module
+
+When you make change to your Move module, there are 2 kinds of changes, compatible and incompatible. You can learn [upgrade policy in details on docs](https://aptos.dev/move/book/package-upgrades/#compatibility-rules). When you make compatible changes, you run `npm run move:upgrade`. When you make incompatible changes, you run `npm run move:publish` to publish to a new address.
```
Comment (hariria) [inline apps/nextra/pages/en/build/create-aptos-dapp/templates/digital-asset.mdx:78]: See Above

File: apps/nextra/pages/en/build/create-aptos-dapp/templates/fungible-asset.mdx [hunk 4]
```diff
@@ -71,6 +71,10 @@ npm run move:init
 npm run move:publish
 ```
 
+### Upgrade the Move module
+
+When you make change to your Move module, there are 2 kinds of changes, compatible and incompatible. You can learn [upgrade policy in details on docs](https://aptos.dev/move/book/package-upgrades/#compatibility-rules). When you make compatible changes, you run `npm run move:upgrade`. When you make incompatible changes, you run `npm run move:publish` to publish to a new address.
+
```
Comment (hariria) [inline apps/nextra/pages/en/build/create-aptos-dapp/templates/fungible-asset.mdx:77]: See Above

**Review by 0xmaayan — APPROVED** [review]

**Review by hariria — APPROVED** [review]

**Pull request merged**